ts/sass/BUILD.bazel

```diff
@@ -18,8 +35,16 @@ sass_library(
     visibility = ["//visibility:public"],
 )
 
+sass_library(
+    name = "scrollbar_lib",
+    srcs = [
+        "scrollbar.scss",
+    ],
+    visibility = ["//visibility:public"],
+)
+
 # qt package extracts colours from source file
 exports_files(
-    ["_vars.scss"],
-    visibility = ["//qt:__subpackages__"],
+    ["_vars.scss"] + glob(["*.scss"], exclude = ["_*.scss"]),
```

Comment:
Do we need this? The pages should be listing the _libs as deps instead of listing the raw .scss files